zerver/openapi/zulip.yaml

```diff
@@ -21143,6 +21143,14 @@ paths:
                   example: false
                 message_retention_days:
                   $ref: "#/components/schemas/MessageRetentionDays"
+                is_archived:
+                  description: |
+                    A boolean indicating whether the channel is [archived](/help/archive-a-channel)
+                    or unarchived.
+
+                    **Changes**: In Zulip 11.0 (feature level {ZF-7ec2f5}).
```

Comment:
Would be good to phrase this as "New in Zulip 11.0" to be consistent with other entries.

Reply:
The curly braces are wrong here; that's not how this substitution system works.

@Vector73 FYI; we probably need to do a pass on documentation ... maybe best is the the `create-api-changelog` tool should print a link to new page in `docs/` about documenting API changes?